TWLight/users/templates/users/editor_detail_data.html

```diff
@@ -66,7 +66,7 @@
             {% trans "Yes" %}
         {% else %}
             {% comment %}Translators: When viewing a user's profile (e.g. https://wikipedialibrary.wmflabs.org/users/ when logged in), this shows if the user does not qualify the technical requirements. {% endcomment %}
-            <p class="bg-danger"><strong class="warning">{% trans "No" %}</strong></p>
+            <p class="bg-danger-soft"><strong class="warning">{% trans "No" %}</strong></p>
```

Comment:
I think you should add the `.twl-links` class to all anchor tags in this template to have a consistent color scheme for links throughout TWL